libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.ts

```diff
@@ -19,8 +19,8 @@ export class ChallengeOverviewComponent {
   organizationCards: OrganizationCard[] = MOCK_ORGANIZATION_CARDS;
   // mockTopics = ['breast', 'cancer'];
 
-  use_default(str: string) {
-    return str === '' ? 'Not available' : str;
+  use_default(str: string | null | undefined) {
+    return str === '' || str == null ? 'Not available' : str;
```

Comment:
Simplify with `return str || 'Not available'`?

Reply:
Another option is to use the nullish coalescing operator (`return str ?? 'Not available'`) IF we consider that an empty string is a valid value. If an empty value is not valid, we should define in the data/API model that a string should have a min length > 0.

Here is a TS playground to learn more about the [nullish coalescing operator](https://www.typescriptlang.org/play?ssl=1&ssc=8&pln=1&pc=35#code/PTAEBUAsFNQOwK4BskEsDOlQGMD2BDJadbVOAc1FwAdoAnfAF1ztA1Hzg6UfridQA3WM1AAfMQCgQoAO6RU2LHWiMEdOOlCMYoOqnKRGAWnSoAJrGgAPaivRncXVADNtuoi5NnL0sO0QUKlYEOEsXMmhzADpJP1AASS48OEYGdEYAGnExUAR0YlAXQnQATxwYbABrdGyAW2hOMkpOUGg66kZS+Iz9CmD3WEQ6gCN6UAAGOVxkc1AxnCcfeiiikuhY+IBBUHJcXDmbfA6iIpZ3dhdGtRU2LUtCZrlUHVBqfDpGVEJ43BGAK2g2EYWnkiiwkHwwlA4XwyBBchgXHwoCq0HKGDgAHJGG98A5VmRNmReHRithYFtqNQAMJOCLkdQCJygADeklAoBkABFoMV4QAuUAAIgAFHBcPBjtAAJTCgDcbQ6XVAvSeCQAyqBBI9zBypQ0hWqKPK4pyeXy4TwhcYAIyKqbsHVoPWcl7tdBC4ZjOim-UW-nW7R0BDQfX4YFCaBCkb7IicU0AXziLlCkZZCGo5iY0Cp1FFKQZQoACh8voQADx5ulwBlMr5OAB8MrZ-rAAHUXlhAkhjHhCMRSP0aPQmCx9YWDNF+A1QABeRa1qcz2AAflXIvFkpXctNnMn5Gi7rqWgXB6PvBPoHXoDte8XDOiEa+0LP9Knz6j143aVDfvNYA0uoKipKquBIAgDZwBO76Hiu87aKUtC4G457wXOGEisa5DCteD7LtKoBCmKEoGrKCowUuh7HqeiHIahsEXh686YcK3r0LhG7njRRG3valGPp+r50dAKH4YeQmwBhC7CrG4GNHAnHiU+kbQkKv7QP+XJgAAqmY-QSFQtAMMwrBgkoiyzLsX4jPgczZow+ACQRs5vlR06EYZJHbtKu7aTCuCFBKuKEEguCyGcrDCrhZDUJBznUZetHcUlOR8YqMjmIFWjBdwYURS45yOnAcWMAlKkvlJymSWlGkZWAKiheUtnmNkhCyPgpRaBpkjJvEACaMw4K0jVzHULCwPgsaQfAyBoJgiwDiQTxkIMoAAMzRAA7PMYWUNQuAZAKfjxEYjDUJ6ICWIIIx7eg0R1IodCHShjDRHgdTAF0tAkPonTAJwEpps0xjfYOf0mOtxhbcAQA).